Fix runtime when deconstructing telecomms mainframes

Funce opened the pull request:
Fixes #23864 

Not very flexible is right.

afterthought2 commented:
I'm not sure that this will work. At any rate, you still won't be able to rebuild it, and will wind up with a useless board. Should add building logic, etc., like for the other machines.

Funce commented:
Ngaaaah this is what I was afraid of. Will do tomorrow.

afterthought2 commented:
Needs a squash. Also the part requirements seem much lower than other tcomms machines.

Funce commented:
I'll add a few more parts that match and do a squash when I get home.

Funce committed:
Fix runtime when deconstructing telecomms mainframes


Create circuitboard so that telecomms mainframe can be rebuilt


Add telecomms onto obj path


Add relevant parts to the mainframe


Fixes snow planet ore maps

They were so poor they didn't pass sanity. Now sanity is more tweakable.

Also fixes lava planet's ultra rich ore map actually being super poor.
